addons/mail/models/mail_template.py

```diff
@@ -244,20 +244,33 @@ def _generate_template(self, res_ids, render_fields):
         """
         self.ensure_one()
 
-        results = dict()
+        render_results = {}
         for _lang, (template, template_res_ids) in self._classify_per_lang(res_ids).items():
-            for field in render_fields:
+            # render fields not rendered by sub methods
+            fields_torender = [
+                field for field in render_fields
+                if field not in ['attachments',  # attachments
+                                 'attachment_ids',  # attachments
+                                 'auto_delete',  # static
```

Comment:
why static ? (means not rendered ?)

maybe we can make only one comment "static", and all the static field below it)

Reply:
Yup, updated comment :)